commands/inits/scripts.go

```diff
@@ -1,29 +1,38 @@
 package inits
 
 import (
-	_ "embed"
+	"os"
+	"embed"
 	"fmt"
 )
 
-//go:embed data/status_shortcuts.sh
-var scriptStatusShortcuts string
+//go:embed data/*
+var scripts embed.FS
```

Comment:
While this mechanism is more convenient for a large number of files, since we're still on a relatively small amount of embedded files, I'd like to keep with the specific embed for each.  It's a bit more unwieldy, however, it causes a missing file to be a _compile-time error_ instead of a _runtime error_, which I think pays dividends in reliability/maintainability.

Reply:
Sure, will change it back. I didn't like the nested `if` statements (wish Go had ternaries...) but your concerns are also valid.

Reply:
I have another idea where I believe we could clean that up a bit, but maybe overkill with just the two options for now.  I'll paste a sample once we get the rest figured out (leaving this convo "unresolved" until then as a reminder to myself).